src/components/html-text-input.js

```diff
@@ -28,6 +28,7 @@ type PropsType = {
 	setTitleAction: string => void,
```

Comment:
The `setTitleAction` doesn't seem to be used anymore, can we remove it?

Reply:
Indeed 👍
(weird that flow didn't report it)